src/__tests__/pages/Selector/index.jsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { render, unmountComponentAtNode } from "react-dom";
 import { act } from "react-dom/test-utils";
 
-import Selector from ".";
+import Selector from "../../../pages/Selector";
 
 let container = null;
```

Comment:
I still need to do further investigation into Jest but in the meantime I think it would be good to wrap any tests for this component in a `describe` function. It keeps test contexts clear and contained when the component grows in complexity.

```
describe('Selector component', () => {
...
});
```